www/install/sql/centstorage/Update-CSTG- 20.10.2.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX `action_object_date` ON `log_action` (`object_type`,`action_log_date`);
```

Comment:
Your work is intended for 3 versions including the 20.04.x.
As you will need to create the same index on the 20.04.x, this will throw an error on platforms upgrading from the 20.04 :
![image](https://user-images.githubusercontent.com/34628915/100984164-9e9eee80-354a-11eb-841a-f7d997997adf.png)

Reply:
![image](https://user-images.githubusercontent.com/34628915/100990290-01e04f00-3552-11eb-8f77-785e4cf211f9.png)

Reply:
@garnier-quentin you should not create the index in 20.04.x branch
Furthemore, it seems the name of the upgrade file contains an empty space

Reply:
Yes you can't create index on 20.04.x. otherwise you'll get an issue on upgrade.